ex4B/ex4Spec.js

```diff
@@ -0,0 +1,12 @@
+var ex4 = require('./ex4.js');
+
+describe("toNumber", function() {
+
+	it("converte the chacter 7 & 1 string  to a number 71", function() {
```

Comment:
Normally, when people say numbers, they mean decimal numbers (https://en.wikipedia.org/wiki/Decimal). So, yes, unless described differently, you should support negative numbers (-3) and decimals with a decimal point (3.14) and, of course, both (-3.14). 
There are many other number notations, but this it the "peoples" default one.